### ISSUE TITLE
500 error

### ISSUE THREAD

**Issue by maxskillpro** [sveltekit-i18n/lib#195]
I use this method, all settings are the same https://github.com/sveltekit-i18n/lib/tree/master/examples/locale-router . The problem is that from a computer the site works fine, but if you go from your phone, every time 500 error. Through dev tools from the phone this error ReferenceError: Cannot access uninitialised variable.
map@[native code]
get_navigation_result_from_branch@http://192.168.0.109:5000/node_modules/@sveltejs/kit/src/runtime/client/client.js:503:37
@http://192.168.0.109:5000/node_modules/@sveltejs/kit/src/runtime/client/client.js:2359:45

**Comment by cam-eo** [comment]
Hey @maxskillpro 

Since this comment was so long ago, I assume that you have figured out your issue.

...But to give clarity:

This is not specific to the example you tried here. 

`get_navigation_result_from_branch` throwing `ReferenceError: Cannot access uninitialized variable` is a long-standing, well-documented SvelteKit/WebKit issue and not specific to this i18n implementation.

There are mixed signals online about the resolution and it might be dependent on the browser and version of that browser you are using on mobile

In short: Issue not caused by `i18n`, it is a known WebKit/JavaScriptCore engine bug and not something this library can fix for you 

Based on this, I will close this comment, but feel free to post a new issue if needed 🙏🏾
